src/core/main.c

```diff
@@ -3284,6 +3286,14 @@ int main(int argc, char *argv[]) {
                 goto finish;
         }
 
+        tag_cache = mnt_new_cache();
```

Comment:
I don't think this belongs here really. Please make it lazily initialized when libmount is actually used, or at least move it into manager_new